integration/test_ml.py

```diff
@@ -34,6 +35,11 @@
 except ImportError:
     _TF_ENABLED = False
 
+try:
+    from google.cloud import automl_v1
```

Comment:
Once this is merged to master, let's also update the `.github/workflows/release.yml` to install this package, and add the Auto ML resource to the project Kobayashi Maru.

Reply:
sounds good